packages/commonwealth/client/scripts/helpers/ContractHelpers/Contest.ts

```diff
@@ -0,0 +1,125 @@
+import { AppError } from '@hicommonwealth/core';
+import { ContestAbi } from './Abi/ContestAbi';
+import ContractBase from './ContractBase';
+import NamespaceFactory from './NamespaceFactory';
+
+class Contest extends ContractBase {
+  namespaceFactoryAddress: string;
+  namespaceFactory: NamespaceFactory;
+
+  constructor(contractAddress: string, factoryAddress: string, rpc: string) {
+    super(contractAddress, ContestAbi, rpc);
+    this.namespaceFactoryAddress = factoryAddress;
+  }
+
+  async initialize(withWallet: boolean = false): Promise<void> {
+    await super.initialize(withWallet);
+    this.namespaceFactory = new NamespaceFactory(
+      this.namespaceFactoryAddress,
+      this.rpc,
+    );
+    await this.namespaceFactory.initialize();
+  }
+
+  /**
+   * deploys a new recurring contest and configures the feeManager to divert feeShare of
+   * stake fees to new contest
+   * @param namespaceName namespace name to configure for
+   * @param contestInterval the recurrng contest interval in ms
```

Comment:
also had question in other PR https://github.com/hicommonwealth/commonwealth/pull/7529#discussion_r1584590532

Reply:
This will be seconds not ms, will update the doc string